x/auth/keeper.go

```diff
@@ -42,6 +43,11 @@ func NewAccountKeeper(
 	}
 }
 
+// Logger returns a module-specific logger.
+func (ak AccountKeeper) Logger(ctx sdk.Context) log.Logger {
+	return ctx.Logger().With("module", "x/auth")
```

Comment:
Maybe a better pattern for module keepers is:

```go
return ctx.Logger().With("module", fmt.Sprintf("x/%s", ModuleName))
```

Reply:
Sure, will make this change.

Reply:
why does auth need to have a `Logger`?